src/resources/i18n/translations/en.json

```diff
@@ -488,7 +488,7 @@
   "No Wifi Connection": "No Wi-Fi connection",
   "No Wifi Only": "No, Wi-Fi only",
   "Notifications": "Notifications",
-  "Notifications UP help text": "Selcet an installed provider (like ntfy) from the dropdown menu to use UnifiedPush. Turn off battery optimization for it to not miss any notifications.",
+  "Notifications UP help text": "Install ntfy, turn off battery optimization for it, and select it here to not miss any notifications,\nUnifiedPush gives push notifications without leaking data.",
```

Comment:
This could be a click to install thing?
Didn't find any alternatives to ntfy for Android, so thought going with that only made sense.
That it is called "UnifiedPush" and presents an alternative to "Push notifications" isn't all that important(?).
The "leaking data" part is the crux here.